src/model/stations.fixtures.ts

```diff
@@ -0,0 +1,1326 @@
+import type { StationModel } from './stations.model'
```

Comment:
I wonder if we can generate the list from a CSV file directly so that when we need to update it, we can just replace the CSV file.

Reply:
@ismailsunni In the final version, only the names of the stations and the order will remain fixed in the code. And the names of the stations and the order should not change in 10/15 years. 
The other information will have to be retrieved by the Rennes web service. The field "desserte" for exemple will get by WebService. 
I think we can we can stay with fixtures instead of csv file